src/lib/web-rpde-query.js

```diff
@@ -0,0 +1,81 @@
+import Settings from './settings.js';
+import { database_pool } from './database.js';
+
+
+
+
+class rpde_query {
+  constructor(afterTimestamp, afterId, limit) {
+    this.afterTimestamp = parseInt(afterTimestamp) > 0 ? parseInt(afterTimestamp) : null;
+    this.afterId = afterId;
+    this.limit = parseInt(limit) > 0 ? parseInt(limit) : Settings.rpdeDefaultLimit;
```

Comment:
Number(limit) would be fine here since we're not parsing something more complex such as `parseInt(34e3, 10)`  which is what I understand that function is really for.

Reply:
ok